3.0-RELEASE-NOTES.md

```diff
@@ -32,4 +32,32 @@ via `global.Promise`,
 you will have to check all places where you are using non-standard promise API
 and update them to use Bluebird API instead.
 
-Please see [Related code change](https://github.com/strongloop/loopback/pull/1896) here.
\ No newline at end of file
+Please see [Related code change](https://github.com/strongloop/loopback/pull/1896) here.
+
+## new method of defining remoting metadata
+
+In 2.0, remote methods were defined as:
+```
+methods: {
+  staticMethod: {
+    isStatic: true,
+    http: { path: '/static' }
+  },
+  instanceMethod: {
+    isStatic: false,
+    http: { path: '/instance' }
+  }
+}
+```
+
+For 3.0, the isStatic flag is no longer required and will be determined from the method name.
+Method name starting with "prototype." will be the same as having isStatic flag set to false.
+```
+methods: {
+  staticMethod: {
+    http: { path: '/static' }
+  },
+  'prototype.instanceMethod': {
+  http: { path: '/instance' }
+}
+```
```

Comment:
Please add a link to this pull request, to make it easier for readers to look up discussions that led to this change. AFAICT, other entries include a link to the relevant pull request too.